### PR TITLE
Add Restify instrumentation package

### PR DESCRIPTION
**Pull request by jquatier**
I created a port of the express middleware that works with [Restify](http://restify.com/). There's just a few APIs that vary between Restify and Express, mainly around what methods are available on the request object. I updated the integration tests to match and they currently pass. The README contains example usage as well. 

Anyway, I'm open to any changes. I could also publish this myself but thought it might belong with the other instrumentations you have available. Let me know! Thanks!

### CODE REVIEW

File: packages/zipkin-instrumentation-restify/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "zipkin-instrumentation-restify",
+  "version": "0.1.0",
```
Comment (eirslett) [inline packages/zipkin-instrumentation-restify/package.json]: verion is 0.2.2 - all the packages in zipkin-js have the same version.

**Comment by eirslett** [comment]
Thanks a lot for the PR!

**Comment by jquatier** [comment]
@eirslett Done. I also updated the top-level README which has the list of packages. I can also put in a PR for the [instrumentations page](https://github.com/openzipkin/openzipkin.github.io/blob/master/_data/existing_instrumentations.yml) on zipkin.io once this package is published.

**Comment by eirslett** [comment]
Thanks a lot!

**Comment by jquatier** [comment]
Great! Let me know when you think this will be available on NPM. Wasn't sure how often you run lerna-publish. In the meantime, I'll update the other repo with a PR.

**Pull request merged**